mne/preprocessing/eyetracking/calibration.py

```diff
@@ -0,0 +1,346 @@
+"""Eyetracking Calibration(s) class constructor."""
+
+# Authors: Scott Huberty <seh33@uw.edu>
+#          Eric Larson <larson.eric.d@gmail>
+#          Adapted from: https://github.com/pyeparse/pyeparse
+# License: BSD-3-Clause
+
+from collections import OrderedDict
+from copy import deepcopy
+
+import numpy as np
+
+from ...utils import _check_fname, fill_doc, logger
+
+
+@fill_doc
+class Calibration(OrderedDict):
+    """Eye-tracking calibration info.
+
+    This data structure behaves like a dictionary. It contains information regarding a
+    calibration that was conducted during an eye-tracking recording.
+
+    .. note::
+        When possible, this class should be instantiated via a helper function,
+        such as :func:`~mne.preprocessing.eyetracking.read_eyelink_calibration`.
+
+    Parameters
+    ----------
+    onset : float
+        The onset of the calibration in seconds. If the calibration was
+        performed before the recording started, then the onset should be
+        set to ``0`` seconds.
+    model : str
+        A string, which is the model of the eye-tracking calibration that was applied.
+        For example ``'H3'`` for a horizontal only 3-point calibration, or ``'HV3'``
+        for a horizontal and vertical 3-point calibration.
+    eye : str
+        The eye that was calibrated. For example, ``'left'``, or ``'right'``.
+    avg_error : float
+        The average error in degrees between the calibration points and the
+        actual gaze position.
+    max_error : float
+        The maximum error in degrees that occurred between the calibration
+        points and the actual gaze position.
+    points : array-like of float, shape ``(n_calibration_points, 5)``
+        The data for the positions, actual gaze, and offsets for each calibration point.
+        Each row should contain data for 1 calibration point. The columns should be
+        of shape ``(5,)`` and contain ``(point_x, point_y, offset, gaze_x, gaze_y)``,
+        where:
+
+            - point_x: the x-coordinate of the calibration point
+            - point_y: the y-coordinate of the calibration point
+            - offset: the error in degrees between the calibration position and the
+                actual gaze position
+            - gaze_x: the x-coordinate of the actual gaze position
+            - gaze_y: the y-coordinate of the actual gaze position
+
+        If the value for a field is not available, use ``np.nan``. See the example below
+        for more details.
+
```

Comment:
I'm surprised NumpyDoc allows you to get away with this extra newline. But at least by local convention we don't put them between params
```suggestion
```